<launch>
    <node name="pid_controller_node1" pkg="pid_controller" type="pid_controller" >
      <remap from="/setpoint" to="/wheel1_velocity"/>
      <remap from="/output" to="/wheel1_output_vel"/>
      <remap from="/PIDController_output" to="/wheel1_velocity_PID"/>
    </node>

    <node name="pid_controller_node2" pkg="pid_controller" type="pid_controller" >
      <remap from="/setpoint" to="/wheel2_velocity"/>
      <remap from="/output" to="/wheel2_output_vel"/>
      <remap from="/PIDController_output" to="/wheel2_velocity_PID"/>
    </node>

    <node name="pid_controller_node3" pkg="pid_controller" type="pid_controller" >
      <remap from="/setpoint" to="/wheel3_velocity"/>
      <remap from="/output" to="/wheel3_output_vel"/>
      <remap from="/PIDController_output" to="/wheel3_velocity_PID"/>
    </node>

    <node name="pid_controller_node4" pkg="pid_controller" type="pid_controller" >
      <remap from="/setpoint" to="/wheel4_velocity"/>
      <remap from="/output" to="/wheel4_output_vel"/>
      <remap from="/PIDController_output" to="/wheel4_velocity_PID"/>
    </node>
</launch>